_migrations/is-migration-assistant-right-for-you.md

```diff
@@ -1,244 +1,57 @@
-| **Documents**                    | Yes                 | Migrate existing data with Reindex-from-Snapshot and live traffic with Capture-and-Replay                                                                                                           |
-| **Index Settings**               | Yes                 | Migrate with Metadata Migration Tool                                                                                                                                                                |
-| **Index Mappings**               | Yes                 | Migrate with Metadata Migration Tool                                                                                                                                                                |
-| **Index Templates**              | Yes                 | Migrate with Metadata Migration Tool                                                                                                                                                                |
-| **Component Templates**          | Yes                 | Migrate with Metadata Migration Tool                                                                                                                                                                |
-| **Aliases**                      | Yes                 | Migrate with Metadata Migration Tool                                                                                                                                                                |
-| **ISM Policies**                 | Expected in 2025    | Manually migrate using API                                                                                                                                                                          |
-| **Elasticsearch Kibana Dashboards** | Expected in 2025 | Only needed if tooling is being used to migrate Elasticsearch Kibana Dashboards to OpenSearch Dashboards. Export JSON files from Kibana and import into OpenSearch Dashboards; before importing, use the [dashboardsSanitizer](https://github.com/opensearch-project/opensearch-migrations/tree/main/dashboardsSanitizer) tool on X-Pack visualizations like Canvas and Lens in Kibana Dashboards, as they may require recreation for compatibility with OpenSearch. |
-| **Security Constructs**          | No                  | Configure roles and permissions based on cloud provider recommendations. For example, if using AWS, leverage IAM for enhanced security management.                                                  |
+| Component | Supported | Recommendations   |
+| :--- |:--- | :--- |
+| **Documents**  | Yes  | Migrate existing data with Reindex from Snapshot (RFS) and live traffic with Capture and Replay. |
```

Comment:
I capitalize elsewhere but there's no set standard, unlike `Reindex-from-Snapshot`, which is how RFS appears in the repo. Lowercasing moving forward.

```suggestion
| **Documents**  | Yes  | Migrate existing data with `Reindex-from-Snapshot` (RFS) and live traffic with capture and replay. |
```